[
    {
        "task_name": "Unnamed task",
        "node": "node",
        "stderr": "",
        "stdout_lines": [
            "The following submodules need updates:",
            "application/adp needs to be set to 0ebb2267738666b5e843bd5bce1b76b292dca8fc",
            "deployment needs to be set to d00531bc67cf4688bac4dc1b184b5f8a2becffce",
            "domains/driving needs to be set to e761ceaf865f9ac85870cbb46ff513d36b5181ab",
            "Visiting domains",
            "Visiting application",
            "Visiting ",
            "New base tree f05f5fc9601e42862fe165d1e14bb35f8e527610",
            "New commit sha: b5d84b7117e91f187956cb1ecabfb951d21855db",
            "Traceback (most recent call last):",
            "  File \"/home/zuul/bump_submodules/bin/speedbump\", line 8, in <module>",
            "    sys.exit(main())",
            "  File \"/home/zuul/bump_submodules/lib/python3.8/site-packages/speedbump/__init__.py\", line 561, in main",
            "    autobump(**vars(args))",
            "  File \"/home/zuul/bump_submodules/lib/python3.8/site-packages/speedbump/__init__.py\", line 526, in autobump",
            "    baserepo.ref('heads/' + branch).update(commit.sha)",
            "  File \"/home/zuul/bump_submodules/lib/python3.8/site-packages/github3/decorators.py\", line 31, in auth_wrapper",
            "    return func(self, *args, **kwargs)",
            "  File \"/home/zuul/bump_submodules/lib/python3.8/site-packages/github3/git.py\", line 234, in update",
            "    json = self._json(self._patch(self._api, data=dumps(data)), 200)",
            "  File \"/home/zuul/bump_submodules/lib/python3.8/site-packages/github3/models.py\", line 156, in _json",
            "    raise exceptions.error_for(response)",
            "github3.exceptions.UnprocessableEntity: 422 Update is not a fast forward"
        ],
        "msg": "non-zero return code",
        "name": []
    }
]